isolate.sh

```diff
@@ -112,11 +112,7 @@ function main() {
         info "Licensing not marked for backup, skipping."
     fi
     restart
-    if [[ $CERT_MANAGER_MIGRATED == "true" ]]; then
-        wait_for_certmanager "$CONTROL_NS" "${ns_list}"
-    else
-        info "Cert Manager not migrated, skipping wait."
-    fi
+    wait_for_certmanager "$CONTROL_NS" "${ns_list}"
```

Comment:
I am testing on a cluster without any CP2 workload installed(no CP2 cert-manager), and doing isolation.
I do not see CP2 cert-manager is installed in `cs-control` namespace. Not sure if we should always wait for cert-manager

Reply:
in wait_for_certmanager, check_if_cert_manager_deployed is called and if no opreq exists in scope, this function will create one so we should always have cert manager deployed to control namespace. https://github.com/IBM/ibm-common-service-operator/blob/3bd2a10da2c5497f66b53085c90e4c37c42adfeb/isolate.sh#L645

Reply:
if there is an opreq that is still in the scope of the master cs ns that requests cert manager, then cert manager will be deployed to cs control. If there is not an opreq within the scope of the master cs ns, then we create an opreq that only requests cert manager in the master cs ns so cert manager will be deployed in cs control

Reply:
Would it be better to not check specifically for ibm-cert-manager-operator in control ns and instead check for `cert-manager-operator` deployment across all namespaces?

Reply:
I was hesitant to `always have cert manager deployed to control namespace`.
But I think it is ok to always have CP2 cert manager in control namespace.

> instead check for cert-manager-operator deployment across all namespaces?

Does this count red hat cert manager? Or it is just looking for ibm cert-manager-operator.